docs_src/microservices/general/messagebus.md

```diff
@@ -175,11 +175,11 @@ The NATS implementations convert the NATS multi-level topic scheme to match that
     
         All events coming from any device service for any device profile, device or source
       
-    - **edgex/events/+/+/camera-001/#**
+    - **edgex/events/+/+/+/camera-001/#**

```

Comment:
Please add example where the device service name is specified.
 I.e. `edgex/events/+/device-onvif-camera/#`

Reply:
Also the two sections `Redis implementation` vs `MQTT 3.1 and NATS implementations` can be collapsed into one now that redis has been updated to match MQTT.

Reply:
updated